cni/network/network_windows.go

```diff
@@ -23,7 +25,17 @@ import (
  * We can delete this if statement once they fix it.
  * Issue link: https://github.com/kubernetes/kubernetes/issues/57253
  */
-func handleConsecutiveAdd(containerId, endpointId string, nwInfo *network.NetworkInfo, nwCfg *cni.NetworkConfig) (*cniTypesCurr.Result, error) {
+func handleConsecutiveAdd(containerId, endpointId, netNs string, nwInfo *network.NetworkInfo, nwCfg *cni.NetworkConfig) (*cniTypesCurr.Result, error) {
+	// Check if the netNs is a valid GUID to decide on HNSv1 or HNSv2
+	if _, err := uuid.Parse(netNs); err == nil {
+		if err = hcn.V2ApiSupported(); err != nil {
+			log.Printf("HNSV2 is not supported on this windows platform")
+			return nil, err
```

Comment:
just return nil, err at the end, if err should be nil anyway if there is no error